tagore

```diff
@@ -16,7 +16,7 @@ import sys
 from argparse import ArgumentParser, HelpFormatter
 
 
-VERSION = '1.0.2'
+VERSION = '1.0.2a'
```

Comment:
Please bump the version to 1.1.0

Reply:
thanks for the fast reply. Sure, just did!

Reply:
Thanks! Yes, that's a good point. An error out might be problematic in cases where a genome really has XXY or other unusual combinations. A warning will be good.